presto-main/src/main/java/com/facebook/presto/ExceededScanLimitException.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto;
+
+import com.facebook.presto.spi.PrestoException;
+import io.airlift.units.DataSize;
+
+import static com.facebook.presto.spi.StandardErrorCode.EXCEEDED_SCAN_LIMIT;
+
+public class ExceededScanLimitException
+        extends PrestoException
+{
+    public ExceededScanLimitException(DataSize limit)
+    {
+        super(EXCEEDED_SCAN_LIMIT, "Exceeded Scan limit of " + limit.toString());
```

Comment:
Let's clarify what does the limit applies to? E.g. whether it applies to amount of data read from storage before or after compression?

Reply:
This is the amount of consumed input bytes read from storage via QueryInfo's getQueryStats().getRawInputDataSize()

Reply:
Concurring @mbasmanova's comment - we should be clear about what scan limit has been exceeded. I will recommend `EXCEEDED_SCAN_RAW_BYTES_READ_LIMIT`. The message can also be made more explicit

Reply:
the message has been fixed to indicate the scan bytes limit has exceeded for this exception